docs/api/session.md

```diff
@@ -353,8 +355,10 @@ app.on('ready', function () {
 
 ## Class: Cookies
 
-The `Cookies` class gives you ability to query and modify cookies. Instances of
-`Cookies` class must be received by using `cookies` property of `Session` class.
+> Query and modify a session's cookies.
+
+Instances of the `Cookies` class must be received by using `cookies` property of
```

Comment:
Maybe `are accessed by` instead of `must be received by`, received read a little weird to me.